### DISCUSSION TITLE
servicemodule.mu

### DISCUSSION THREAD

**Discussion by zer0Kerbal** [zer0Kerbal/MIR#61] [Q&A] [answered]
@Toa-Sampa 

seems we have one model floating around that currently doesn't have a part.cfg

* ServiceModule.mu

It seems to use the following textures (not included either)
* SDHI_2.5_ServiceModule_texture.png
* SDHI_2.5_ServiceModule_bump_NRM.png

**Comment by Toa-Sampa** [comment, marked as answer]
I think you can remove that.  Sdhi is another mod that is semi-active
